x-pack/plugins/maps/common/descriptor_types/map_descriptor.ts

```diff
@@ -41,6 +42,7 @@ export type Goto = {
 export type TooltipFeature = {
   id: number;
   layerId: string;
+  mbProperties?: GeoJsonProperties | null;
```

Comment:
`GeoJsonProperties` allows for null so this can just be `mbProperties?: GeoJsonProperties;`

Reply:
It does indeed. but see above comment. We're actually normalizing this so clients (potentially _not_ Maps-app, since this is public API), don't need to care.